R/Nepal_SEES.R

```diff
@@ -0,0 +1,23 @@
+#'
+#'
+#' SEES Typhoid data
+#'
+#' A subset of data from the SEES project highlighting Typhoid longitudinal
+#' data from Nepal.
+#'
+#' @format ## `nepal_sees`
+#' A [base:data.frame()] with 904 rows and 8 columns:
+#' \describe{
+#'  \item{Country}{Country name}
+#'  \item{person_id}{ID identifying a study participant}
+#'  \item{sample_id}{ID identifying sample taken}
+#'  \item{bldculres}{Pathogen participant tested positive for; Typhoid or
+#'  paratyphoid}
+#'  \item{antigen_iso}{The antigen/antibody combination included in the assay}
+#'  \item{studyvisit}{Categorical estimated time frame for when sample was
+#'  taken; 28 days, 3_months, 6_months, 12_months, baseline, or 18_months}
+#'  \item{dayssincefeveronset}{Continuous measurement showing how exact days
+#'  since symptom onset}
+#'  \item{result}{Continuous variable describing ELISA result}
+#' }
+#' @source Link to SEES study?
```

Comment:
You just need one more line here:
```suggestion
#' @source Link to SEES study?
"nepal_sees"
```
see https://r-pkgs.org/data.html#sec-documenting-data

Reply:
Added "nepal_sees" to document.